datumaro/plugins/yolo_format/importer.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+from typing import Any, Dict, List
+
+import yaml
+
+from datumaro import Importer
+from datumaro.components.format_detection import FormatDetectionContext
+from datumaro.plugins.yolo_format.extractor import (
+    Yolo8Extractor,
+    Yolo8ObbExtractor,
+    Yolo8PoseExtractor,
+    Yolo8SegmentationExtractor,
+)
+from datumaro.plugins.yolo_format.format import Yolo8PoseFormat
+
+
+class YoloImporter(Importer):
+    @classmethod
+    def detect(cls, context: FormatDetectionContext) -> None:
+        context.require_file("obj.data")
+
+    @classmethod
+    def find_sources(cls, path) -> List[Dict[str, Any]]:
+        return cls._find_sources_recursive(path, ".data", "yolo")
+
+
+class Yolo8Importer(Importer):
+    EXTRACTOR = Yolo8Extractor
+
+    @classmethod
+    def detect(cls, context: FormatDetectionContext) -> None:
+        context.require_file("data.yaml")
```

Comment:
https://github.com/cvat-ai/datumaro/pull/44#discussion_r1636028926

Reply:
The name of the configuration file can now be passed as an argument when exporting/importing yolo8.
If the name is not passed, the importer tries to select it itself - data.yaml, if present, or any file.yaml, if there is only one in the target folder.